src/prog_models/prognostics_model.py

```diff
@@ -1147,7 +1147,9 @@ def generate_surrogate(self, load_functions, first_output = None, threshold_keys
             'inputs': self.inputs,
             'outputs': self.outputs,
             'events': self.events,
-            'stability_tol': 1e-05
+            'stability_tol': 1e-05,
+            'data_add_noise': True,
```

Comment:
for future implementations we may think of giving the option to add noise to the states only and not the input.
for example:
'state_add_noise': True,
'input_add_noise': False,
so that the noise is not automatically added to the input if not necessary.
However, I'm not convinced should go in right away. we need maybe a little more testing.

Reply:
I think this is a great point. Between this suggestion and our other discussions about specifying individual state noise, etc., I think we can improve this in the future.